pkgs/applications/editors/manuskript/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, zlib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
```

Comment:
is this used as only an application, or also as a Python library? In case of the former, please use `buildPythonApplication`.